app/views/summaries/index.html.erb

```diff
@@ -8,16 +8,20 @@
   <h2><%= "#{@patron.first_name} #{@patron.last_name}" %></h2>
   <dl class="row">
     <% if @patron.patron_type == 'Fee borrower' %>
-      <dt class='col-2 col-sm-1 font-weight-normal'>Type</dt>
+      <dt class='col-2 col-sm-1 font-weight-normal'>Type:</dt>
       <dd class='col-10 col-sm-11 patron-type'><%= @patron.patron_type %></dd>
     <% end %>
-    <dt class="col-2 col-sm-1 font-weight-normal">Status</dt>
+    <dt class="col-2 col-sm-1 font-weight-normal">Status:</dt>
     <dd class="col-10 col-sm-11 patron-status"><%= @patron.status %></dd>
     <% if @patron.borrow_limit %>
-      <dt class="col-2 col-sm-1 font-weight-normal">Borrower limit</dt>
+      <dt class="col-2 col-sm-1 font-weight-normal">Borrower limit:</dt>
       <dd class="col-10 col-sm-11 patron-status"><%= pluralize(@patron.borrow_limit, 'checkout') %></dd>
     <% end %>
-    <dt class="col-2 col-sm-1 font-weight-normal">Email</dt>
+    <% if @patron.expired_date %>
```

Comment:
Ditto @jvine https://github.com/sul-dlss/mylibrary/pull/222#issuecomment-514417024
I remember talking to @shelleydoljack and @saseestone about this. It seems like there is more turnover in student priv expiration dates due to enrollment issues, so displaying the expiration date for student users causes unnecessary headaches. 

Could you add an explicit check here that a patron is either a fee borrower or a proxy borrower?